test/test_files/ib_flat/ib_flat.i

```diff
@@ -0,0 +1,81 @@
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#            SIMULATION STOP            #
+#.......................................#
+time.stop_time               =   -10.0     # Max (simulated) time to evolve
+time.max_step                =   300          # Max number of time steps
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#         TIME STEP COMPUTATION         #
+#.......................................#
+time.fixed_dt         =   -0.05        # Use this constant dt if > 0
+time.cfl              =   0.45         # CFL factor
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#            INPUT AND OUTPUT           #
+#.......................................#
+time.plot_interval            =  100       # Steps between plot files
+time.checkpoint_interval      =  -1       # Steps between checkpoint files
+
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
+#               PHYSICS                 #
+#.......................................#
+ConstValue.density.value = 1.0
+ConstValue.velocity.value = 10.0 0.0 0.0
+
+io.outputs = ib_levelset ib_normal ib_src_term
+
+incflo.velocity = 30.0 0.0 0.0
+incflo.use_godunov = 1
+incflo.godunov_type = "weno_z"
+incflo.do_initial_proj = 1
+incflo.initial_iterations = 3
+ICNS.source_terms = IBForcing #BodyForce
+#BodyForce.magnitude = 1.0 0 0
+transport.viscosity = 1.0e-5
+transport.laminar_prandtl = 0.7
+transport.turbulent_prandtl = 0.3333
+turbulence.model = Laminar
+
+incflo.physics = FreeStream IB
+IB.labels = IB1  
+#IB.IB1.type =  Hill 
+#IB.IB1.center = 0.0 0.0 0.0
+#IB.IB1.height = 0.04
+#IB.IB1.half_width = 0.1
+IB.IB1.type =  Flat
+IB.IB1.height = 0.1
+IB.IB1.drag_coefficient = 0.1
+IB.IB1.eps_spread = 0.3
```

Comment:
I would think it would be good to have all our complex terrain parameters under the ParmParser `IB.ComplexTerrain`.  This would segregate cases where we've implemented the model vs the ones we haven't (cylinder, etc). @gdeskos what are your thoughts? it looks like the parser isn't setup that way.

Reply:
@psakievich  this is an idea I have been battling with for some time. The idea behind the label is that you are creating an object (e.g. IB1) and then you assign a type which in this case is the Flat terrain, so that you can add more than one IBs (for example one hills and one box) and then compute forces and everything else separately. Also this allows you to specialize and read input according to the type. If we do IB.ComplexTerrain then you can't add different IB objects

Reply:
It seems like we should be able to do this with a few minor changes. The actuator stuff works in a way where you can assign the same parameter at different scopes.  In terms of reading the input you can still make specialized inputs, you just need to partition the parsing into a general function and a specialized function.  Let's not worry about it for now though. This is not really a critical feature. It is just slightly more work on the user for now. We can perfect the interface after we prove the method.